cmd/kots/cli/install.go

```diff
@@ -222,6 +240,10 @@ func InstallCmd() *cobra.Command {
 	cmd.Flags().String("config-values", "", "path to a manifest containing config values (must be apiVersion: kots.io/v1beta1, kind: ConfigValues)")
 	cmd.Flags().Bool("port-forward", true, "set to false to disable automatic port forward")
 	cmd.Flags().String("wait-duration", "2m", "timeout out to be used while waiting for individual components to be ready.  must be in Go duration format (eg: 10s, 2m)")
+	cmd.Flags().String("http-proxy", "", "sets HTTP_PROXY environemnt variable in kotsadm containers")
```

Comment:
(applies to all 4):

- change spelling of environment.
- replaced `kotsadm containers` with "all KOTS Admin Console components"